File: app/src/main/java/gateway/hitrontech/com/encryption/utils/WhichContext.java
package gateway.hitrontech.com.encryption.utils;

import android.app.Application;
import android.content.Context;

public class WhichContext extends Application {

  private static Context mContext;

  static Context getInstance() {
    return mContext;
  }

  @Override
  public void onCreate() {
    super.onCreate();
    WhichContext.mContext = getApplicationContext();
  }
}

File: app/src/main/java/gateway/hitrontech/com/encryption/utils/WhichContext.kt
package gateway.hitrontech.com.encryption.utils

import android.annotation.SuppressLint
import android.app.Application
import android.content.Context

class WhichContext : Application() {

    override fun onCreate() {
        super.onCreate()
        WhichContext.instance = applicationContext
    }

    companion object {

        @SuppressLint("StaticFieldLeak")
        var instance: Context? = null
            private set
    }
}
